test.js

```diff
@@ -1,2 +1,9 @@
 console.log('test.js')
-coconuy()
\ No newline at end of file
+coconuy()
+
+var x = 10
+var y = 5
+if x > y 
+  console.log("x is greater than y")
+else 
+  console.log("x is less than or equal to y")
```

Comment:
:robot: OpenAI

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [8-10]

The indentation in lines 8 and 10 is incorrect. The code inside the `if` and `else` blocks should be indented with spaces or tabs. For example:

```javascript
if (x > y) {
    console.log("x is greater than y");
} else {
    console.log("x is less than or equal to y");
}
```

11:
There is an unnecessary newline at the end of the file. It can be removed.

<!-- This is an auto-generated comment by OpenAI -->